roles/bwc/tasks/bwc_repos_apt.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Assert that master_token is specified
+  fail: msg="License key must be supplied for BWC enterprise installation."
```

Comment:
Not needed in this file, since you include that check in previous step `bwc_repos_setup.yml`